src/examples/wasm/callback/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+	<meta charset="utf-8" />
+	<title>Go WebAssembly</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<script src="wasm_exec.js" defer></script>
+	<script src="wasm.js" defer></script>
+</head>
+
+<body>
+	<h1>WebAssembly</h1>
+	<p>Add two numbers, using WebAssembly:</p>
+	<input type="number" id="a" value="0" /> + <input type="number" id="b" value="0" /> = <input type="number"
+		id="result" readonly />
```

Comment:
It's a trivial thing, but maybe have the result input all be on one line?

Reply:
Fixed. Yes that's a bit unclean. (Probably some unintended auto-wrapping by my editor?)